Intacct/ReadMe.md

```diff
@@ -1,33 +1,33 @@
-![Kingdom First Solutions](https://user-images.githubusercontent.com/81330042/113314137-9628ab80-92d1-11eb-9541-5f95c2ba1d92.png)
-
-
+![Kingdom First Solutions](../.screenshots/KFSBanner.jpg)
 
 # Intacct Export to Journal 
-_Tested/Supported in Rock Version:  8.0-13.0_    
+_Tested/Supported in Rock Version:  13.0-14.0_    
 _Released:  10/9/2018_   
-_Updated:  2/2/2022_   
-
-
+_Updated:  4/13/2023_   
 
 ## Summary 
 
 This plug in will allow you to create journal entries in Intacct for batches with the click of a button.
 
 Quick Links:
-
 - [What's New](#whats-new)
 - [Configuration](#configuration)
+    - [Intacct Configuration](#intacct-configuration)
+    - [Rock Configuration](#rock-configuration)
 - [Advanced Configuration](#advanced-configuration)
 
 ## What's New
 
 The following new goodness will be added to your Rock install with this plugin:
 
+- **New Page**: Intacct Export (Finance > Functions > Intacct Export)  
 - **New Block**: Batch to Journal (added to the Batch Detail Page on install)  
+- **New Block**: Batches to Journal (added to the new Intacct Export Page on install)  
 - **New Account Attributes**: There are a number of new Account attributes that control where transactions are posted in Intacct  
 - **New Page**: Intacct Projects (Finance > Administration > Intacct Projects)  
```

Comment:
Let's try to keep like items together, so moving Intacct Projects New Page up to the top.
```suggestion
- **New Page**: Intacct Projects (Finance > Administration > Intacct Projects)  
- **New Page**: Intacct Export (Finance > Functions > Intacct Export)  
- **New Block**: Batch to Journal (added to the Batch Detail Page on install)  
- **New Block**: Batches to Journal (added to the new Intacct Export Page on install)  
- **New Account Attributes**: There are a number of new Account attributes that control where transactions are posted in Intacct  
```